.secrets.baseline

```diff
@@ -0,0 +1,149 @@
+{
+  "exclude": {
+    "files": null,
+    "lines": null
+  },
+  "generated_at": "2020-10-23T21:00:22Z",
+  "plugins_used": [
+    {
+      "name": "AWSKeyDetector"
+    },
+    {
+      "name": "ArtifactoryDetector"
+    },
+    {
+      "base64_limit": 4.5,
+      "name": "Base64HighEntropyString"
+    },
+    {
+      "name": "BasicAuthDetector"
+    },
+    {
+      "name": "CloudantDetector"
+    },
+    {
+      "hex_limit": 3,
+      "name": "HexHighEntropyString"
+    },
+    {
+      "name": "IbmCloudIamDetector"
+    },
+    {
+      "name": "IbmCosHmacDetector"
+    },
+    {
+      "name": "JwtTokenDetector"
+    },
+    {
+      "keyword_exclude": null,
+      "name": "KeywordDetector"
+    },
+    {
+      "name": "MailchimpDetector"
+    },
+    {
+      "name": "PrivateKeyDetector"
+    },
+    {
+      "name": "SlackDetector"
+    },
+    {
+      "name": "SoftlayerDetector"
+    },
+    {
+      "name": "StripeDetector"
+    },
+    {
+      "name": "TwilioKeyDetector"
+    }
+  ],
+  "results": {
+    "docs/openapi.yaml": [
+      {
+        "hashed_secret": "f9fdc64928c96c7ad56bf7da557f70345d83a6ed",
+        "is_verified": false,
+        "line_number": 1472,
+        "type": "Base64 High Entropy String"
+      }
+    ],
+    "main.go": [
+      {
+        "hashed_secret": "5baa61e4c9b93f3f0682250b6cf8331b7ee68fd8",
+        "is_verified": false,
+        "line_number": 31,
+        "type": "Basic Auth Credentials"
+      }
+    ],
+    "vendor/github.com/jmoiron/sqlx/go.sum": [
```

Comment:
ugh these vendor files should also be excluded. but i guess we can do that later

Reply:
Is there some reference for what should and should not be included in .secrets.baseline?

Reply:
i don't think so, just common sense - lock files are always false positives because they contain hashes
it's fine to keep the vendors dir but not the most useful thing to scan 🤷‍♀️

Reply:
Gotcha. Seen several repos with the .locks in the secrets baseline so wasn't sure if there were more subtle reasons

Reply:
Nah, probably just laziness :p it's fine to include these files but it's annoying because it makes the baseline file very long. When a new secret is added to the baseline we should check whether it's a real secret we should remove, and a long file with a bunch of false positives makes it easier to miss real secrets